### ISSUE TITLE
HTML tag displayed in the search results

### ISSUE THREAD

**Issue by shantanu156** [o19s/quepid#782]
**Describe the bug**
Not sure if a bug. Please close if unnecessary. 
For the sub-field description whenever there is a match, I see the field displayed with the HTML tags like below example:
![image](https://github.com/o19s/quepid/assets/62331655/858735ea-4192-4825-b985-925a3752e83c)

**Expected behavior**
It should represent simply normal without the tags and the matched word is bold.

Is this expected or anything else it needed to be done? I didn't see a solution anywhere, hence reporting here.

**Comment by epugh** [comment]
Yeah, the whole logic around the rendering of nested docs like this is a bit wonky.   We preserve the structure of the data by making it JSON, which then means we don't render the HTML.       WOuld the right thing be to render each line as HTML?    We DO have logic in the book of judgements to actually render the rows....
